view/desktop/bin/src/lib.rs

```diff
@@ -6,26 +6,24 @@ mod utl;
 mod window;
 
 pub mod constants;
+mod cli;
 
 use platform_core::context_v2::ContextCell;
 use platform_core::platform::cross::client::CrossPlatformClient;
 use platform_workspace::WorkspaceId;
 use rand::random;
 use std::env;
-use std::path::PathBuf;
 use std::rc::Rc;
 use std::sync::Arc;
-use homedir::{my_home};
-use tauri::{AppHandle, Manager, RunEvent, WebviewUrl, WebviewWindow, WindowEvent};
+use tauri::{AppHandle, Manager, RunEvent, WebviewWindow, WindowEvent};
+use tauri_plugin_cli::CliExt;
 use tauri_plugin_log::{fern::colors::ColoredLevelConfig, Target, TargetKind};
 use window::{create_window, CreateWindowInput};
 use workbench_desktop::window::{NativePlatformInfo, NativeWindowConfiguration};
 use workbench_desktop::Workbench;
-
-
+use crate::cli::cli_handler;
 use crate::commands::*;
 use crate::constants::*;
-use crate::plugins as moss_plugins;
```

Comment:
This should not be removed, as this import is used in a block of code specific to macOS. If it is removed, a compilation error will occur on macOS.

<img width="629" alt="image" src="https://github.com/user-attachments/assets/af76685d-825b-4c3b-8b1d-f0056d62ae58">

Reply:
Sorry I didn't notice that! I guess I shouldn't have blindly trusted cargo check lol

Reply:
This is probably because this code doesn’t compile with your compiler, and cargo check doesn’t catch it. I’ll show you a trick later on how to avoid these issues